sci-geosciences/gmt/gmt-4.5.18-r1.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Powerful map generator"
+HOMEPAGE="https://gmt.soest.hawaii.edu/"
+SRC_URI="
+	mirror://gmt/${P}-src.tar.bz2
+	gmttria? ( mirror://gmt/${P}-non-gpl-src.tar.bz2 )"
+
+LICENSE="GPL-2 gmttria? ( Artistic )"
```

Comment:
Verify the license, please.

Reply:
This line is also from the old ebuilds. Main code license is GPL 2 or later (old ebuilds are incoorect), so I changed GPL-2 on GPL-2+. Triangulation code is distributed under its own license (GPL-like, as I understand it) and I do not know how to specify this in the ebuild.

Reply:
So it's not `Artistic`? (compare the text with `licenses/Artistic` in the repo)

Reply:
No, texts are different